dpnp/dpnp_iface_mathematical.py

```diff
@@ -637,7 +637,9 @@ def sum(x1, axis=None, **kwargs):
 
     is_x1_dparray = isinstance(x1, dparray)
 
-    if (not use_origin_backend(x1) and is_x1_dparray and not kwargs):
+    if (not use_origin_backend(x1) and is_x1_dparray and len(kwargs) <= 1):
+        axis = kwargs.pop('axis') if len(kwargs) == 1 else None
```

Comment:
Just to beautify this line of code
https://docs.python.org/3/library/stdtypes.html#dict.pop

Reply:
Perhaps https://docs.python.org/3/library/stdtypes.html#dict.get is applicable here